safe/gui/tools/shake_grid/shakemap_converter_dialog.py

```diff
@@ -228,14 +240,28 @@ def accept(self):
         # noinspection PyUnresolvedReferences
         QtGui.qApp.setOverrideCursor(QtGui.QCursor(QtCore.Qt.WaitCursor))
 
+        extra_keywords = {}
+        if self.check_box_custom_shakemap_id.isChecked():
+            event_id = self.line_edit_shakemap_id.text()
+            extra_keywords[extra_keyword_earthquake_event_id['key']] = event_id
+
+        current_index = self.combo_box_source_type.currentIndex()
+        source_type = self.combo_box_source_type.itemData(current_index)
+        LOGGER.debug(current_index)
+        LOGGER.debug(source_type)
```

Comment:
Just a question, if we let these debug statements, can we have more descriptive informations? When someone send us some debug message, it make it difficult to debug when only something like `yes`, `3`, `false`, `tab changed` :D

Reply:
oops, it means to be deleted. Personal use only :D

Reply:
We don't have a clear policy about these debug messages. If it's debug level, I don't mind if we have a kind of sentence to introduce the variable